### DISCUSSION TITLE
Passenger bots teamkill me in little bird chopper

### DISCUSSION THREAD

**Discussion by Digitalinsite** [Joe91/fun-bots#262] [General]
hi, first of all - AWESOME MOD! seriously. I went out and bought BF3 premium because I saw this mod. Thanks guys  - absolutely epic work. 

Bug I found is that if I am flying in the little bird and bots spawn as passengers, they often teamkill me. I'm guessing they are firing at enemy NPC's and that the bots don't have the locked fov range like a player does when they are a passenger in the bird. I have friendly fire on BTW.

Looking forward to your future updates. You guys rock.

**Comment by Joe91** [comment]
Thanks for that feedback. Will have to turn on teamkill for myself to have a look at this :-) . 
Do they kill you with the primary weapon?

**Comment by Digitalinsite** [comment]
Yeah it does sound like its with whatever rifle they have. Thanks again

**Comment by Firjens** [comment]
https://github.com/Joe91/fun-bots/issues/263

**Comment by Joe91** [comment]
may I ask what version you are using and on what map you reproduce this?
Had a few tries without the bots firing at all...
I used the 3rd person camera in the little bird to proof that the passengers can't move further than the default-positions....
![grafik](https://user-images.githubusercontent.com/6923271/203854077-c112357e-66ec-4b2a-a8b8-87fd10840b16.png)

**Comment by Digitalinsite** [comment]
hey Joe, 

I just tried it now on Markaz Monolith Conquest 64 and had a bot Teamkill me when I flew low over an enemy-controlled conquest point.  I had about 3 bots on the sides of the chopper with assault rifles and a sniper and one of them popped me in the head. 

Server settings below

mapvote.excludecurrentmap true
mapvote.limit 60
mapvote.randomize true
vars.bulletDamage 170
vars.friendlyfire true
vars.gameModeCounter 150 
vars.hud false
vars.idleTimeout 0
vars.killCam true
vars.maxPlayers 64
vars.miniMap false
vars.nameTag disable
vars.nameTag false
vars.onlySquadLeaderSpawn disabled
vars.playerManDownTime 200
vars.playerManDownTime 200
vars.regenerateHealth false
vars.soldierHealth 100 
vars.teamKillCountForKick 100
vars.teamKillCountForKick 50
vars.teamKillValueDecreasePerSecond 0
vars.teamKillValueForKick 100
vars.teamKillValueForKick 4
vars.teamKillValueIncrease 1
vars.vehicleSpawnDelay 150
vu.ColorCorrectionEnabled false
vu.DisablePreRound true
vu.SetTeamTicketCount 1 300
vu.SetTeamTicketCount 2 300
vu.SunFlareEnabled false
vu.SuppressionMultiplier 15
vu.TimeScale 1.1
vu.VehicleDisablingEnabled false

I do have the "Vehicle-Modifications-main" mod which changes the FOV in vehicles though. https://github.com/VU-PINK/Vehicle-Modifications  Don't think it does anything else that would effect bots. Pretty sure I've had the bots teamkill me on Operation Firestorm when I'm in the big chopper too. 

Using Funbots latest version ("Version": "2.6.2") and I've tweaked the bots aim FOV in the settings to 80.

Thanks heaps

**Reply by Joe91** [reply]
Thank you for all that information!
Can you try to reproduce it once without the other mods and if it still happens then on the new dev-version of fun-bots?
I will try to reproduce it on the other version as well but it maybe really might be the vehicle-fev-mod...

**Comment by Digitalinsite** [comment]
Absolutely. I'll try on that map with only Funbots running. Easy fix for now is to turn off friendly fire :)

Thanks - I will get back to you.
